libs/common/utilities/src/css/layout/padding/stories/vl-padding.stories.ts

```diff
@@ -14,77 +15,88 @@ export default {
 };
 
 const paddingCss = css`
-    .block {
+    .sb-container {
         width: 300px;
         height: 100px;
         background-color: lightblue;
-        padding: var(--vl-spacing--medium) 0; /* wordt via vl-region gezet */
+        border: rgb(152, 191, 34) 5px solid;
+        padding: 15px; /* normaal specifieert de container (bvb. de vl-section) dit */
     }
 
-    .block__inner {
+    .sb-content {
         width: 100%;
         height: 100%;
-        background-color: mediumspringgreen;
-        border: lightseagreen 2px solid;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-size: 14px;
+        background-color: white;
+        border: black 1px dashed;
     }
 `;
 
 export const PaddingDefault = ({}) => html`
     <style>
         ${paddingCss}
     </style>
-    <div class="block">
-        <div class="block__inner">&nbsp;default&nbsp;</div>
+    <div class="sb-container">
+        <div class="sb-content">default</div>
     </div>
 `;
 PaddingDefault.storyName = 'vl-padding - default';
+PaddingDefault.parameters = formattedSourceCode;
 
 export const PaddingSmall = ({}) => html`
     <style>
         ${paddingCss}
     </style>
```

Comment:
Ik vind de code voorbeelden & stories al heel wat duidelijker!

Ik zou mss wel verduidelijken dat de groene rand het element is waarop de `vl-padding-next` klasse is toegepast. Ik had het niet direct door :) Maar toen ik het door had vond ik het wel duidelijk! (en dan hetzelfde bij de `vl-margin-next`). Ik zou het zelfs overwegen dat in een Alert te plaatsen.

Reply:
kijk naar het tekeningetje bovenaan, dat is expliciet hetzelfde groen en ook met een stippellijn

Reply:
Ok. Ik denk nog steeds dat het geen kwaad kan om dat expliciet te vermelden, maar door de plaats van die afbeelding had ik die link nog niet gelegd - als je dat weet is het idd wel duidelijker. Maar dat weet ik nu ook enkel omdat jij met hebt gezegd :)

We kunnen het laten voor wat het is